pyproject.toml

```diff
@@ -13,14 +13,8 @@ python = "^3.8"
 uhashring = "^2.1"
 marisa-trie = "^1.0.0"
 
-[tool.poetry.dev-dependencies]
-flake8 = "^5"
-flake8-annotations = "^2.9"
-flake8-bandit = "^4.1"
-flake8-black = "^0.3"
-flake8-bugbear = "^22"
-flake8-docstrings = "^1.6"
-mypy = "^0"
```

Comment:
Were these supposed to be removed?

Reply:
Yes, we install those on nox sessions, so this was pretty much unused. We don't need to freeze flake or mypy versions. The puristic way would be to freeze them so the builds are reproducible, but for such a small project I think pinning those caused more noise than good.

If you feel strongly, I can re-add those and remove the installs from nox, change to poetry for the dev dependencies. But we were basically not using this.